content/infrastructure/installation/install-rippled-on-ubuntu.md

```diff
@@ -91,6 +89,28 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
         sudo setcap 'cap_net_bind_service=+ep' /opt/ripple/bin/rippled
 
 
+9. Optional: configure core dumps
+
+    By default Ubuntu is not configured to produce core files useful for debugging crashes.
+    First run:
+    
+            ulimit -c unlimited
```

Comment:
For consistency and to avoid extraneous spaces in the code block when parsed, this should be indented the same as the other example (4 spaces more than the surrounding paragraph)

```suggestion
        ulimit -c unlimited
```